src/components/dashboard/dashboardItem/ItemBase.js

```diff
@@ -133,7 +133,14 @@ const DashboardItem = ({ item }) => {
                 </Typography>
             </CardContent>
 
-            <CardActions>{item.actions}</CardActions>
+            <div
```

Comment:
GitHub Actions is complaining about the `CardActions` import no longer being used.